include/simdjson/document.h

```diff
@@ -373,13 +373,13 @@ class document {
   bool dump_raw_tape(std::ostream &os) const noexcept;
 
   /** @private Structural values. */
-  std::unique_ptr<uint64_t[]> tape;
+  std::unique_ptr<uint64_t[]> tape = {};
```

Comment:
Why are some of these `= {}` and and some of them `{}`?

Reply:
It is semantically equivalent but let us standardize.